gnd/src/main/java/com/google/android/gnd/persistence/local/LocalValueStore.java

```diff
@@ -121,4 +122,14 @@ public boolean isTermsOfServiceAccepted() {
   public void setTermsOfServiceAccepted(boolean value) {
     preferences.edit().putBoolean(TOS_ACCEPTED, value).apply();
   }
+
+  /** Returns whether the polygon info dialog was previously shown to the user or not. */
+  public boolean isPolygonDialogInfoShown() {
+    return preferences.getBoolean(POLYGON_INFO_DIALOG, false);
+  }
+
+  /** Updates the polygon info dialog value to stop showing the dialog everytime. */
+  public void setPolygonInfoDialogShown(boolean value) {
```

Comment:
You will also want to clear this when the user signs out.

Reply:
updated.